builder/dockerfile/parser/testfiles/escapes/Dockerfile

```diff
@@ -6,9 +6,7 @@ RUN apt-get \update && \
 ADD \conf\\" /.znc
 
 RUN foo \
-
 bar \
-
```

Comment:
since you've remove all of the tests for "blank lines" as continutations, let's make sure we add a test to ensure that a blank line end the continution. E.g. Make sure that:
```
RUN foo \

bar
```
gives an error of "unknown command 'bar'`